packages/loader/container-loader/src/connectionManager.ts

```diff
@@ -594,6 +594,14 @@ export class ConnectionManager implements IConnectionManager {
 					LogLevel.verbose,
 				);
 			} catch (origError: any) {
+				this.logger.sendTelemetryEvent(
+					{
+						eventName: "ConnectToDeltaStreamException",
+						connected: connection !== undefined && connection.disposed === false,
```

Comment:
Wouldn't you want these logged separately so you can really determine what's up?  I'd have done this:

```ts
details: { defined: connection !== undefined, disposed: connection.disposed }.
```

It'll get logged as JSON so you can see all the cases.